src/components/Sidebar.js

```diff
@@ -33,16 +33,42 @@ const StyledA = styled(A)`
   }
 `
 
+const LiveDot = styled.span`
+  height: 10px;
+  width: 10px;
+  background-color: ${p => p.theme.colors.text};
+  border-radius: 50%;
+  margin: 0 7px 0 4px;
+  display: inline-block;
+`
+
+const LiveLabel = styled.p`
+  margin: 1em 0 2em 60px;
+  font-weight: 600;
+  font-size: .9em;
+  border-radius: 7px;
+  background-color: ${p => p.theme.colors.primary};
+  color: ${p => p.theme.colors.secondaryBackground};
+  width: 4em;
+  padding: 5px;
+`
+
 export default () => {
   const [location] = useLocation();
 
   return (
     <SidebarContainer>
-      <Header>Some Title</Header>
+      <Header>nwHacks</Header>
```

Comment:
Thoughts on having this as a constant in `Constants.js`

Reply:
Should we eventually put this in the cms? If yes I will leave this for now?

Reply:
Oh yeah, we should meet to talk about Livesite x CMS integration LOL I guess we can leave it as is rn